Bottom: 8d08fbeb30ab915cfac4714d71272fa1134ce238
Top:    f174a380458e4f1c86221cacbd6e0ba7efe55559
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-29 14:13:37 -0500

Refresh of builder-add-new-host

---
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index fa054a69..8058586d 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -1,3 +1,5 @@
+builder.novatech-llc.com ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
+
 [bacula]
 testdaedalus.novatech-llc.com
